gazelle/common/treesitter/BUILD.bazel

```diff
@@ -10,9 +10,9 @@ go_library(
     importpath = "aspect.build/cli/gazelle/common/treesitter",
     visibility = ["//visibility:public"],
     deps = [
+        "//gazelle/common/treesitter/grammars/kotlin",  #keep
```

Comment:
These need `#keep` because gazelle is disabled in `gazelle/common/treesitter/grammars/BUILD.bazel`